### DISCUSSION TITLE
how use REST APIs Upload single file

### DISCUSSION THREAD

**Discussion by chaochaokeji** [nocodb/nocodb#1010] [Discord Threads]
**Please enter the following details**

Copy and paste project info : [Youtube Video](https://www.youtube.com/watch?v=AUSNN-RCwhE)

or 

please provide the following details :
NocoDB used as docker : true / false
NocoDB version : 
Database used in NC_DB URL  : mysql | pg | mssql | sqlite3 / (defaults to sqlite3 if empty)
Project was created  by clicking : New Project | New Project by connecting to external database
Database on which spreadsheet is created : mysql | pg | mssql | sqlite3 / (defaults to sqlite3 if empty)
OS on which NocoDB is running : 
Node.js version if running as node : 
Database version : 

**Steps To Reproduce**
3. See error

**Expected behavior**

**Screenshots**

Join our discord : https://discord.gg/5RgZmkW for realtime help.

Node version : **v12.22.1**
Arch type: **x64**
Platform: **linux**
Docker: **true**
Database: **sqlite3**
packageVersion: **0.9.27**

**Comment by o1lab** [comment]
Moving this to discussion

**Comment by o1lab** [comment]
https://github.com/nocodb/nocodb/discussions/369
